README.md

```diff
@@ -1,9 +1,9 @@
 # Bottlerocket Admin Container
 
-This is the default admin container for [Bottlerocket](https://github.com/bottlerocket-os/bottlerocket).
-The admin container has an SSH server that lets you log in as ec2-user using your EC2-registered SSH key.
-It also runs agetty services for serial console devices to allow console access.
-It runs outside of Bottlerocket's container orchestrator in a separate instance of containerd.
+This is the default admin container for troubleshooting [Bottlerocket](https://github.com/bottlerocket-os/bottlerocket), which runs in a separate instance of `containerd`.
```

Comment:
nit: the wording here is ambiguous and it could be read as saying "Bottlerocket runs in a separate instance of `containerd`"

Reply:
Ah, I can see that. I'll post a revision and maybe make it more consistent with the control container description as well.